src/main/setupInvestHandlers.js

```diff
@@ -166,3 +136,25 @@ export function setupInvestRunHandlers(investExe) {
     });
   });
 }
+
+export function setupInvestLogReaderHandler() {
+  ipcMain.on(ipcMainChannels.INVEST_READ_LOG, async (event, logfile, channel) => {
+    const fileStream = fs.createReadStream(logfile);
+    fileStream.on('error', (err) => {
+      logger.error(err);
+      event.reply(
+        `invest-stdout-${channel}`,
+        `Logfile is missing or unreadable: ${os.EOL}${logfile}`
+      );
+    });
+
+    const rl = readline.createInterface({
+      input: fileStream,
+      crlfDelay: Infinity,
```

Comment:
Well this is an interesting option!  I wonder what their use case was for expecting there to be an observable delay between `\r` and `\n`.

Reply:
I guess it must have happened at some point! Based on the docs `Infinity` seems like a reasonable cross-platform choice without any downside in our case.